pyaedt/modules/solutions.py

```diff
@@ -1071,338 +1075,248 @@ def ifft_to_file(
 
 
 class FfdSolutionData(object):
-    """Contains Hfss Far Field Solution Data (ffd)."""
+    """Contains information from the far field solution data.
+    Load far field data from the element pattern files.
```

Comment:
```suggestion
    
    Load far field data from the element pattern files.
```
Blank line before supplying additional details about the class. Since the class doesn't do the action, do you think rewriting to this might be better? "Far field solution data is loaded from element pattern files."